start_bs.py

```diff
@@ -105,11 +109,18 @@ def filter_camera_data(camera):
     vdcm = VerticalDCM('XF:17IDA-OP:AMX{Mono:DCM', name='vdcm')
     zebra = Zebra('XF:17IDB-ES:AMX{Zeb:2}:', name='zebra')
     vector_program = VectorProgram('XF:17IDB-ES:AMX{Gon:1-Vec}', name='vector_program')
-else:
+    robot = EMBLRobot()
+elif beamline == "fmx":  
     mercury = ABBIXMercury('XF:17IDC-ES:FMX{Det:Mer}', name='mercury')
     mercury.read_attrs = ['mca.spectrum', 'mca.preset_live_time', 'mca.rois.roi0.count',
                                             'mca.rois.roi1.count', 'mca.rois.roi2.count', 'mca.rois.roi3.count']
     vdcm = VerticalDCM('XF:17IDA-OP:FMX{Mono:DCM', name='vdcm')
     zebra = Zebra('XF:17IDC-ES:FMX{Zeb:3}:', name='zebra')
     vector_program = VectorProgram('XF:17IDC-ES:FMX{Gon:1-Vec}', name='vector_program')
 
+    robot = EMBLRobot()
+elif beamline == "nyx":
+    denso_robot = Robot("XF:19IDC-ES{Rbt:1}", name="robot")
+    robot = DensoRobot(denso_robot)
```

Comment:
These names are a bit confusing, it'd be better to distinguish between the ophyd classes/instances and the robot API classes/instances.

Reply:
completely agree - will work on updating the Robot class name in mxtools and then clean up the code downstream from there

Reply:
commit https://github.com/NSLS-II/lsdc/pull/102/commits/a53cacbad93d418692ec663ac1e2e8dc27ce887f - in nyxtools, have renamed the Robot class to DensoOphydRobot, given more descriptive names to the objects, and added a note about the DensoRobot, which is the robot_lib API-compatible object. hope this meets with your satisfaction.